docs/customization.rst

```diff
@@ -244,15 +189,11 @@ Configuration Options
             settings. Although this icon can be `any of the icons bundled with this theme`_,
             popular choices are:
 
-            - |fa-git| ``fontawesome/brands/git``
-            - |fa-git-alt| ``fontawesome/brands/git-alt``
-            - |fa-git-square| ``fontawesome/brands/git-square``
-            - |fa-github| ``fontawesome/brands/github``
-            - |fa-github-alt| ``fontawesome/brands/github-alt``
-            - |fa-github-square| ``fontawesome/brands/github-square``
-            - |fa-gitlab| ``fontawesome/brands/gitlab``
-            - |fa-gitkraken| ``fontawesome/brands/gitkraken``
-            - |fa-bitbucket| ``fontawesome/brands/bitbucket``
+            .. jinja:: repo_icons
+
+                {% for icon in repo_icons %}
```

Comment:
Can just use:

```
{% set repo_icons = ... %}
```

that way the icons can be specified inline rather than in `conf.py`.

Reply:
Or it might work to just specify the list inline in the `{% for`